backend/src/main/java/org/eclipse/tractusx/puris/backend/erpadapter/ErpAdapterConfiguration.java

```diff
@@ -40,4 +41,28 @@ public class ErpAdapterConfiguration {
      */
     @Value("${puris.erpadapter.authsecret}")
     private String erpAdapterAuthSecret;
+
+    @Value("${puris.erpadapter.refreshinterval}")
+    @Getter(AccessLevel.NONE)
+    private long refreshInterval;
+
+    @Value("${puris.erpadapter.timelimit}")
+    @Getter(AccessLevel.NONE)
+    private long refreshTimeLimit;
+
+    /**
+     * @return The refresh time limit in milliseconds
+     */
+    public long getRefreshTimeLimit() {
```

Comment:
During discussion you called that threshold which was easier to understand for me. time limet and refresh interval are relatively close to each other. First reading I considered this might be a timeout.